Hyper/Window.cs

```diff
@@ -1,92 +1,35 @@
-﻿using Character.Shaders;
-using Chunks.ChunkManagement;
-using Chunks.MarchingCubes;
-using Common.UserInput;
-using Hud;
-using Hud.Shaders;
-using Hyper.Controllers;
-using Hyper.Shaders;
+﻿using Common;
 using NLog;
-using OpenTK.Graphics.OpenGL4;
 using OpenTK.Windowing.Common;
 using OpenTK.Windowing.Desktop;
 using OpenTK.Windowing.GraphicsLibraryFramework;
 
 namespace Hyper;
 
-internal class Window : GameWindow, IInputSubscriber
+public class Window : GameWindow
 {
-    private static readonly Logger Logger = LogManager.GetCurrentClassLogger();
-
-    private Thread _console = null!;
-
-    private Scene _scene = null!;
-
-    private IController[] _controllers = null!;
+    private Game _game;
 
-    private readonly Context _context = Common.UserInput.Context.Instance;
+    private static readonly Logger Logger = LogManager.GetCurrentClassLogger();
 
     public Window(GameWindowSettings gameWindowSettings, NativeWindowSettings nativeWindowSettings)
         : base(gameWindowSettings, nativeWindowSettings)
     {
-        StartDebugThreadAsync();
-
-        RegisterCallbacks();
+        _game = new Game(nativeWindowSettings.Size.X, nativeWindowSettings.Size.Y, new WindowHelper(this), Guid.NewGuid().ToString());
```

Comment:
Maybe use current time instead of GUID? It'd be super nice if the user could specify that as well